bundles/org.openhab.binding.openwebnet/NOTICE

```diff
@@ -0,0 +1,20 @@
+This content is produced and maintained by the openHAB project.
+
+* Project home: https://www.openhab.org
+
+== Declared Project Licenses
+
+This program and the accompanying materials are made available under the terms
+of the Eclipse Public License 2.0 which is available at
+https://www.eclipse.org/legal/epl-2.0/.
+
+== Source Code
+
+https://github.com/openhab/openhab2-addons
+
+== Third-party Content
+
+Modified version of rx-openwebnet
```

Comment:
Can you provide the sources of this modified version?
Seems the only way people can now modify/maintain this binding is by [decompiling the library](https://community.openhab.org/t/bticino-openwebnet-new-openhab2-binding-ready-for-testing/43377/1049?u=wborn).

Reply:
The binding is now based on openwebnet4j lib bundle, available here: https://github.com/mvalla/openwebnet4j